webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,21 @@ function BugForm() {
     }, this));
   };
 
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand", function() {
+      var savedValue = this.value;
+      this.value = "";
+      this.baseScrollHeight = this.scrollHeight;
+      this.value = savedValue;
+    })
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      var minRows = this.getAttribute("data-min-rows") | 0, rows;
```

Comment:
I have removed the bitwise OR and added different values for `minRows` for comment section and form section.